lib/Mojo/Reactor/Epoll.pm

```diff
@@ -82,8 +65,10 @@ sub remove {
 		}
 		warn "-- Removed IO watcher for $fd\n" if DEBUG;
 		return !!delete $self->{io}{$fd};
-	} else {
+	} elsif (my $timer = $self->{timers}{$remove}) {
 		warn "-- Removed timer $remove\n" if DEBUG;
+		$self->remove($timer->{fd});
+		delete $self->{zero_timers}{$remove};
```

Comment:
This seems to be the only instance of `$self->{zero_timers}`

Reply:
Oh right that's a left-over from an older version